gbif/pipelines/ingest-gbif-java/src/main/java/org/gbif/pipelines/ingest/java/pipelines/VerbatimToInterpretedPipeline.java

```diff
@@ -192,6 +194,12 @@ public static void run(InterpretationPipelineOptions options, ExecutorService ex
             .create()
             .counterFn(incMetricFn);
 
+    GrscicollTransform grscicollTransform =
+        GrscicollTransform.builder()
+            .kvStoreSupplier(GrscicollLookupKvStoreFactory.getInstanceSupplier(config))
+            .create()
+            .counterFn(incMetricFn);
```

Comment:
To make it work you must add metric to org.gbif.pipelines.ingest.java.metrics.IngestMetricsBuilder